docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/lib/components/cgdp-wide-guide-card/index.scss

```diff
@@ -29,6 +29,7 @@
 	}
 	.nci-guide-card__title {
 		@include typeset("heading", "xl");
+		@include u-line-height('heading', 2);
```

Comment:
The line-height should go into `typeset()` as the third option. 

Additionally, I wonder if the token for the line-height should be used (`$theme-heading-line-height`). The use of the token would be for the case in which the line-height of this heading should match the line height of prose. If that were the case, then this should probably use one of the heading tokens too (`$theme-h2-font-size`).

@andyvanavery31 - Is the intent that the font size of the Wide Guide Card should be the same as a visual/semantic H2 in prose? (Whereby if we globally changed the font-size of body content, this should change as well)

Reply:
@bryanpizzillo @karlikpj the intent is that the line-height for the WGC title is consistent with other headings of the same size, which is the same as nci-heading-h2 (font size: 24.96px and line-height: 29.952px). Yes it would be good to use a token for the font so that in the future if we made a change to global size it would apply to this.